### PR TITLE
Multiple endpoints

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: doc/release-notes-0.12.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+This page discusses the changes that you need to be aware of when migrating your Ziti deployment from version 0.11.x to version 0.12.x
+
+# Theme
+Ziti 0.12 includes the following: 
+ 
+ * Terminators have been extracted from services
+     * Terminators configurate where a service terminates. Previously each service had exactly one terminator. Now services can have 0 to N terminators.
```
Comment (scareything) [inline doc/release-notes-0.12.md]: Terminators _configure_? (instead of _configurate_)

File: doc/release-notes-0.12.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+This release includes the following:
+
+  * Terminators extracted from service as separate entities
+  * When SDK applications bind and unbind, the controller now dynamically adds/removes terminators as appropriate
+ 
+This release does not yet include a terminator strategy API. Strategies can be specified per service, but if a service has multiple terminators the first one will be used. The terminator strategy API along with some implementations will be added in a follow-up release. This release also does not include strategy inputs on terminators as discussed in the above design document. If strategy inputs end up being useful, they may be added in the furure.
+
+## Terminator related API changes
+
+There is a new terminators endpoint:
```
Comment (scareything) [inline doc/release-notes-0.12.md]: terminators endpoint! lol

File: doc/release-notes-0.12.md (hunk 1 above)
Comment (plorenz) [reply doc/release-notes-0.12.md]: Fixed :)

File: doc/release-notes-0.12.md (hunk 2 above)
Comment (plorenz) [reply doc/release-notes-0.12.md]: Almost as good as endpoints enpoint :)

File: doc/release-notes-0.12.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+This page discusses the changes that you need to be aware of when migrating your Ziti deployment from version 0.11.x to version 0.12.x
+
+# Theme
```
Comment (andrewpmartinez) [inline doc/release-notes-0.12.md]: I really like the format of these as they have developed.

**Pull request merged**